src/main/java/com/sprint/mission/discodeit/controller/AuthController.java

```diff
@@ -0,0 +1,33 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.request.LoginRequest;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.service.AuthService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import io.swagger.v3.oas.annotations.tags.Tags;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@RequiredArgsConstructor
+@Controller
+@ResponseBody
+@RequestMapping("/api/auth")
+@Tag(name = "Auth", description = "인증 API")
+public class AuthController {
+
+  private final AuthService authService;
+
+  @RequestMapping(path = "login", method = RequestMethod.POST)
```

Comment:
`@PostMapping`을 사용해볼까요?
```suggestion
  @PostMapping(value = "login")
```